src/compiler/diagnosticMessages.json

```diff
@@ -2869,6 +2869,10 @@
         "category": "Error",
         "code": 6143
     },
+    "A package for '{0}' was found at '{1}', but is untyped. Because '--noImplicitAny' is enabled, this package must have a declaration.": {

```

Comment:
how about something more consistent with the rest of the the `--noImplicitAny` error message, e.g.: `Could not find a declaration file for module '{0}'. All imports implicitly have type 'any'.`